src/loader-components/ExampleTemplate.module.scss

```diff
@@ -49,10 +50,27 @@
     display: flex;
     outline: none;
     padding: 0.625rem;
+
+    &.is-selected {
+      background-color: var(--terra-dev-site-example-template-selected-code-toggle-background-color, #666565);
```

Comment:
Do we need to add theme variables for this? If so, we need to add values for the other themes. Is this something we could log an issue and add in later?